packages/@aws-cdk/aws-lambda-python/test/integ.function.poetry.py27.ts

```diff
@@ -0,0 +1,30 @@
+/// !cdk-integ pragma:ignore-assets
+import * as path from 'path';
+import { Runtime } from '@aws-cdk/aws-lambda';
+import { App, CfnOutput, Stack, StackProps } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as lambda from '../lib';
+
+/*
+ * Stack verification steps:
+ * aws lambda invoke --function-name <function name> --invocation-type Event --payload $(base64 <<<'"OK"') response.json
+ */
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const fn = new lambda.PythonFunction(this, 'my_handler', {
```

Comment:
Can you please consolidate all these integration tests into a single test with multiple `PythonFunction`s?

Reply:
Done!